### PR TITLE
tmp walkaround for legacy files in mongo quaratine

### PR DESCRIPTION
**Pull request by kastoestoramadus**
https://jira.tools.tax.service.gov.uk/browse/FILE-499

### CODE REVIEW

File: project/MicroService.scala [hunk 1]
```diff
@@ -86,7 +86,10 @@ trait MicroService {
     .settings(
       resolvers += Resolver.bintrayRepo("hmrc", "releases"),
       resolvers += Resolver.jcenterRepo
-    )
+    ).dependsOn(sharedFileUploadBackend)
+
+  lazy val sharedFileUploadBackend =
+    ProjectRef(uri("https://github.com/hmrc/file-upload-shared.git#master"), "fileUploadShared")
```
Comment (kastoestoramadus) [inline project/MicroService.scala]: External dependency

File: app/uk/gov/hmrc/fileupload/quarantine/QuarantineRepository.scala [hunk 2]
```diff
@@ -36,7 +36,7 @@ import scala.concurrent.Await
 import scala.concurrent.duration._
 import scala.util.{Failure, Success}
 
-case class FileData(length: Long = 0, filename: String, contentType: Option[String], data: Enumerator[Array[Byte]] = null)
+// TMP case class FileData(length: Long = 0, filename: String, contentType: Option[String], data: Enumerator[Array[Byte]] = null)
```
Comment (kastoestoramadus) [inline app/uk/gov/hmrc/fileupload/quarantine/QuarantineRepository.scala]: This version was pushed as is to external dep

File: project/MicroService.scala (hunk 1 above)
Comment (kastoestoramadus) [inline project/MicroService.scala]: After the review the tagged or released version should be targeted.

File: app/uk/gov/hmrc/fileupload/frontendGlobal.scala [hunk 3]
```diff
@@ -103,6 +110,17 @@ class ApplicationModule(context: Context) extends BuiltInComponentsFromContext(c
   lazy val fileUploadController =
     new FileUploadController(redirectionFeature, withValidEnvelope, inMemoryBodyParser, commandHandler, uploadToQuarantine, createS3Key, now)
 
+  // TMP
```
Comment (codesurf42) [inline app/uk/gov/hmrc/fileupload/frontendGlobal.scala]: more descriptive - maybe replace it with your commit message

**Pull request merged**